### ISSUE TITLE
organization for effects in mixer

### ISSUE THREAD

**Issue by tjingboem** [kunstmusik/blue#432]
some thoughts about the organization of the effects in Blue.
In Linux Mint i always check the option 'always On Top' to be sure that the effects don't "disappear" when i go and work in the score. Because of this always-on-top function, i have a lot of screen clutter that consists of effect swimming around the screen.

Possible solutions:
- button on every mixerstrip that, when activated, shows the effects for that channel. It must be possible to push more than 1 button in 1 strip
- rack system like Reason
- rack system like Ableton

**Comment by tjingboem** [comment]
Some further thoughts on the organization of effects:
working with effects and being it quite messy at the moment, i wondered if it were possible to append the instrID to the effect in the mixerstrip.
This is how it looks like now:
![effectmess](https://user-images.githubusercontent.com/6670911/57128653-6777c280-6d94-11e9-9dc5-f239a85f00f0.png)
There are 3 instances of the same effect on the different mixerstrips. Mixerstrips themselves are based on the instrument ID. But what effect belongs to which strip??

Is there a way to add the instrID to each effect so the user knows which effect belongs to which channel?
Perhaps there is another solution to this issue?

**Comment by kunstmusik** [comment]
Interesting, I think I don't run into this as I haven't used so many
effects like this myself. I think maybe something like:

[1] - loopIt!
[Master] - loopIt!
[Subchannel1] - loopIt!

Might not be perfect but would at least be an improvement for the current
design and I think it wouldn't be too tricky to implement.

**Comment by tjingboem** [comment]
yes, i use a lot of effects; this has to do with that i do not make a real distinction between instrument and effect. It all comes down to the sounding result in looking for textures and timbres, and the effect influences the final result.

As a side note, i took a look at Reaper and BitWig and how they organize effects. Reapers' style reminds me of how 30-year old Cubase handles effects, while BitWig has come up with a new style that might be better for Blue and Csound:
![BitWig](https://user-images.githubusercontent.com/6670911/57139191-db29c780-6db4-11e9-910f-61f896da67f7.png)
Here only the effects belonging to the active track are shown. A flaw might be that only the latest active track shows its effects, it can't show ALL effects for ALL the active tracks.

Your solution of [1] - loopIt! etc.  would be great and prevent confusion and definitely be an improvement.
